### PR TITLE
[Zurich] Use new LV95/CH1903+ geocoder and base map tiles

### PR DESCRIPTION
**Pull request by davea**
- Use new `getLocation95` method for geocoding addresses
 - Replace Coordinates::CH1903 with Coordinates::CH1903Plus
 - Refactor Map::Zurich to use WMTSBase instead of duplicating code
 - Slightly refactor map templates to make using custom WMTS tiles easier
 - Use correct sizes for marker pins on maps
 - Disable geocoder cache on staging sites

### CODE REVIEW

**Comment by davea** [comment]
This is now ready for review.

File: perllib/FixMyStreet/Geocode/Zurich.pm [hunk 1]
```diff
@@ -67,7 +67,7 @@ sub string {
     my $cache_dir = FixMyStreet->config('GEO_CACHE') . 'zurich/';
     my $cache_file = $cache_dir . md5_hex($s);
     my $result;
-    if (-s $cache_file && -M $cache_file <= 7) {
+    if (-s $cache_file && -M $cache_file <= 7 && !FixMyStreet->config('STAGING_SITE')) {
```
Comment (dracos) [inline perllib/FixMyStreet/Geocode/Zurich.pm]: Want to do this to the general cache function too?

**Comment by dracos** [comment]
:+1:

File: perllib/FixMyStreet/Geocode/Zurich.pm (hunk 1 above)
Comment (davea) [inline perllib/FixMyStreet/Geocode/Zurich.pm]: Good idea, have done so.

**Pull request merged**